packages/ember-console/lib/index.js

```diff
@@ -1,3 +1,14 @@
+import { deprecate } from 'ember-debug';
+
+// Deliver message that the function is deprecated
+let deprecateEmberLogger = function(){
```

Comment:
I definitely appreciate the desire to be a bit DRY here, but the current implementation has a larger footprint in production builds (where the deprecation is stripped). Would you mind inlining the deprecation in each of the methods instead (I am sorry that we can’t have nice things...)?

Reply:
See if you like the approach I just checked in - I've split the difference to ensure that repeatedly typing long strings can't result in unnecessary and hard-to-spot differences, while still calling deprecate clearly in each method.